api/v1beta1/types.go

```diff
@@ -263,3 +263,17 @@ func (s *HCloudNetworkSpec) IsZero() bool {
 	}
 	return true
 }
+
+// HCloudBootState defines the boot state of an HCloud server during provisioning.
```

Comment:
it's not only "during provisioning" but in general. I think we should remove "during provisioning".